src/Sulu/Bundle/PageBundle/Resources/config/serializer/Document.BasePageDocument.xml

```diff
@@ -6,6 +6,8 @@
         <property name="creator" type="integer" groups="defaultPage,preview"/>
         <property name="changer" type="integer" groups="defaultPage,preview"/>
         <property name="created" type="DateTime" groups="defaultPage,preview"/>
+        <property name="lastModifiedEnabled" type="boolean" groups="defaultPage"/>
```

Comment:
We are not writing a property instead return a virtual property based on our new `getLastModifiedEnabled` method:

```
        <virtual-property method="getLastModifiedEnabled" name="lastModifiedEnabled" serialized-name="lastModifiedEnabled" groups="defaultPage"/>
 ```